test/supports.test.js

```diff
@@ -155,4 +155,12 @@ describe('process()', () => {
     supports.process(rule)
     expect(rule.params).toEqual('(color black) and not ((-moz-a: 1) or (a: 1))')
   })
+
+  it("shouldn't throw errors", () => {
+    let rule = { params: ' (selector( :nth-child(1n of a, b) )) or (c: b(1)) ' }
```

Comment:
Can you add also a case without white-space between `selector(` and `:`?

Reply:
I have added one more test.